packages/core/src/routes/connector.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines */
```

Comment:
nit: try to split files

Reply:
Add LOG-4890 to track this, and will do this in a later cooldown cycle.